sdk/identity/azure-identity/inc/azure/identity/environment_credential.hpp

```diff
@@ -12,37 +12,61 @@
 #include <azure/core/credentials/token_credential_options.hpp>
 
 #include <memory>
+#include <string>
+#include <vector>
 
 namespace Azure { namespace Identity {
+  /**
+   * @brief Options for token authentication.
+   *
+   */
+  struct EnvironmentCredentialOptions final : public Core::Credentials::TokenCredentialOptions
+  {
+    /**
+     * @brief For multi-tenant applications, specifies additional tenants for which the credential
+     * may acquire tokens. Add the wildcard value `"*"` to allow the credential to acquire tokens
+     * for any tenant in which the application is installed.
+     */
+    std::vector<std::string> AdditionallyAllowedTenants;
+  };
+
   /**
    * @brief Environment Credential initializes an Azure credential, based on the system environment
    * variables being set.
    *
+   * @note May read from the following environment variables:
+   * - `AZURE_TENANT_ID`
+   * - `AZURE_CLIENT_ID`
+   * - `AZURE_CLIENT_SECRET`
+   * - `AZURE_CLIENT_CERTIFICATE_PATH`
+   * - `AZURE_CLIENT_CERTIFICATE_PASSWORD`
+   * - `AZURE_CLIENT_SEND_CERTIFICATE_CHAIN`
+   * - `AZURE_USERNAME`
+   * - `AZURE_PASSWORD`
+   * - `AZURE_AUTHORITY_HOST`
    */
   class EnvironmentCredential final : public Core::Credentials::TokenCredential {
   private:
     std::unique_ptr<TokenCredential> m_credentialImpl;
 
+    explicit EnvironmentCredential(
+        Core::Credentials::TokenCredentialOptions const& options,
+        std::vector<std::string> const& additionallyAllowedTenants);
+
   public:
     /**
      * @brief Constructs an Environment Credential.
-     *
      * @param options Options for token retrieval.
-     *
-     * @note May read from the following environment variables:
-     * - AZURE_TENANT_ID
-     * - AZURE_CLIENT_ID
-     * - AZURE_CLIENT_SECRET
-     * - AZURE_CLIENT_CERTIFICATE_PATH
-     * - AZURE_CLIENT_CERTIFICATE_PASSWORD
-     * - AZURE_CLIENT_SEND_CERTIFICATE_CHAIN
-     * - AZURE_USERNAME
-     * - AZURE_PASSWORD
-     * - AZURE_AUTHORITY_HOST
      */
     explicit EnvironmentCredential(
-        Azure::Core::Credentials::TokenCredentialOptions options
-        = Azure::Core::Credentials::TokenCredentialOptions());
+        Core::Credentials::TokenCredentialOptions const& options
+        = Core::Credentials::TokenCredentialOptions());
```

Comment:
Is changing an argument from a value type to a const reference type not considered a breaking change?

Reply:
It is not: https://godbolt.org/z/d3f5PTsYE